src/main/scala/viper/gobra/backend/ViperBackends.scala

```diff
@@ -119,7 +119,7 @@ object ViperBackends {
     /** returns an existing ViperCoreServer instance or otherwise creates a new one */
     protected def getOrCreateServer(config: Config)(executionContext: GobraExecutionContext): ViperCoreServer = {
       server.getOrElse({
-        var serverConfig = List("--disablePlugins", "--logLevel", config.logLevel.levelStr)
```

Comment:
interesting, I wasn't aware we were explicitly disabling plugins in ViperServer

Reply:
Since we ran the plugins (or at least some of them) before passing the AST to the backends, there was no need to run them again